config/config.go

```diff
@@ -1690,3 +1690,16 @@ func (c *Config) UpdateConfig(configPath string, newCfg *Config, dryrun bool) er
 func GetConfig() *Config {
 	return &Cfg
 }
+
+// RemoveExchange removes an exchange config
+func (c *Config) RemoveExchange(exchName string) bool {
+	m.Lock()
+	defer m.Unlock()
+	for x := range c.Exchanges {
+		if strings.EqualFold(c.Exchanges[x].Name, exchName) {
+			c.Exchanges = append(c.Exchanges[:x], c.Exchanges[x+1:]...)
```

Comment:
Might be better to rewrite as this so it can be garbage collected as the struct fields have pointers:
```go
			c.Exchanges[x] = c.Exchanges[len(c.Exchanges)-1]
			c.Exchanges[len(c.Exchanges)-1] = ExchangeConfig{}
			c.Exchanges = c.Exchanges[:len(c.Exchanges)-1]
```